probe_test.go

```diff
@@ -626,3 +626,49 @@ func TestLicenseStatus(t *testing.T) {
 		t.Fatalf("metric compare: err %v", err)
 	}
 }
+
+func TestLinkStatus(t *testing.T) {
+	c := newFakeClient()
+	c.prepare("api/v2/monitor/system/link-monitor", "testdata/link-monitor.jsonnet")
+	r := prometheus.NewPedanticRegistry()
+	if !testProbe(probeLinkMonitor, c, r) {
+		t.Errorf("probeLinkMonitor() returned non-success")
+	}
+
+	em := `
+# HELP fortigate_link_bandwidth_rx Bandwidth available on this link for sending
+# TYPE fortigate_link_bandwidth_rx gauge
+fortigate_link_bandwidth_rx{link="wan1",link_group="wan-mon",vdom="root"} 33557
+# HELP fortigate_link_bandwidth_tx Bandwidth available on this link for sending
+# TYPE fortigate_link_bandwidth_tx gauge
+fortigate_link_bandwidth_tx{link="wan1",link_group="wan-mon",vdom="root"} 68657
+# HELP fortigate_link_jitter electronical jitter on this link
+# TYPE fortigate_link_jitter gauge
+fortigate_link_jitter{link="wan1",link_group="wan-mon",vdom="root"} 1.1268666982650757
+# HELP fortigate_link_latency_seconds latency of this link in seconds
+# TYPE fortigate_link_latency_seconds gauge
+fortigate_link_latency_seconds{link="wan1",link_group="wan-mon",vdom="root"} 0.006810200214385986
+# HELP fortigate_link_packet_loss_total number of packets lost on this link
+# TYPE fortigate_link_packet_loss_total counter
+fortigate_link_packet_loss_total{link="wan1",link_group="wan-mon",vdom="root"} 0
+# HELP fortigate_link_packet_received_total Number of packets received on this link
+# TYPE fortigate_link_packet_received_total counter
+fortigate_link_packet_received_total{link="wan1",link_group="wan-mon",vdom="root"} 278807
+# HELP fortigate_link_packet_sent_total Number of packets sent on this link
+# TYPE fortigate_link_packet_sent_total counter
+fortigate_link_packet_sent_total{link="wan1",link_group="wan-mon",vdom="root"} 278878
+# HELP fortigate_link_sessions Number of sessions active on this link
+# TYPE fortigate_link_sessions gauge
+fortigate_link_sessions{link="wan1",link_group="wan-mon",vdom="root"} 77
```

Comment:
_active_sessions maybe would be better to signal that it is the current value as opposed to some cumulative count.

Reply:
good idea